src/Tizen.WebView/Tizen.WebView/BackForwardList.cs

```diff
@@ -0,0 +1,133 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+
+namespace Tizen.WebView
+{
+    /// <summary>
+    /// This class provides the properties of Back Forward list item of a specific WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class BackForwardListItem
+    {
+        private IntPtr _item_handle;
+
+        internal BackForwardListItem(IntPtr handle)
+        {
+            _item_handle = handle;
+        }
+
+        public string GetURL()
+        {
+            return Interop.ChromiumEwk.ewk_back_forward_list_item_url_get(_item_handle);
+        }
+
+        public string GetTitle()
+        {
+            return Interop.ChromiumEwk.ewk_back_forward_list_item_title_get(_item_handle);
```

Comment:
Did you test this code?
``` c
 const char* ewk_back_forward_list_item_title_get(const Ewk_Back_Forward_List_Item* item);
```
it return a const char*, and it should not be released by caller. 
but, if you use auto marshing with string it will release a pointer after convert to managed
https://stackoverflow.com/questions/6300093/why-cant-i-return-a-char-string-from-c-to-c-sharp-in-a-release-build

Reply:
That's a good point!
@BasavarajPS 
Plz refer to ewk_settings_default_text_encoding_name_get case.

Reply:
Done